src/index.ts

```diff
@@ -84,6 +84,23 @@ const buildTimeTraitsArray = <FactoryResultType>(
   return Array.isArray(traits) ? traits : [traits];
 };
 
+const getValueOrOverride = (
+  overrides: Overrides<any>,
+  traitOverrides: Overrides<any>,
+  fieldValue: any,
+  fieldKey: string
+): any => {
```

Comment:
We don't want the `any` types here, we need to make this function have accurate types.

Reply:
What do you think about something like this?

```
const getValueOrOverride = <FactoryResultType> (
  overrides: Overrides<FactoryResultType>,
  traitOverrides: Overrides<FactoryResultType>,
  fieldValue: Field,
  fieldKey: string
): Field => {
```

Reply:
Yeah, I think that looks reasonable.